pom.xml

```diff
@@ -95,7 +95,7 @@
       at SonarSource/sonarqube/gradle.properties
     -->
     <sonar.version>10.6.0.92116</sonar.version>
-    <sonar.api.version>10.7.0.2191</sonar.api.version>
+    <sonar.api.version>10.11.0.2468</sonar.api.version>
```

Comment:
Small comment: It would be good to use the actual version number instead of "latest version" in the ticket name/commit, so that things are clearer.